src/pages/components/icon.tsx

```diff
@@ -0,0 +1,151 @@
+            description={t('basicIcons.description')}
+            exampleFirst
+          >
+            <ComponentExample style={{ justifyContent: 'flex-start' }}>
+              <Icon icon="check" />
+            </ComponentExample>
+          </ComponentDescription>
+
+          <ComponentDescription
+            mainTitle={t('infoIcons.title')}
+            description={t('infoIcons.description')}
+            exampleFirst
+          >
+            <ComponentExample
+              style={{
+                justifyContent: 'flex-start',
+              }}
+            >
+              <Icon
+                icon="helpFilled"
+                color={suomifiDesignTokens.colors.highlightBase}
+                style={{ margin: `0 ${suomifiDesignTokens.spacing.s}` }}
+              />
+              <Icon
+                icon="help"
+                color={suomifiDesignTokens.colors.highlightBase}
+                style={{ margin: `0 ${suomifiDesignTokens.spacing.s}` }}
+              />
+            </ComponentExample>
+          </ComponentDescription>
+
+          <ComponentDescription
+            mainTitle={t('additionalInfoIcons.title')}
+            description={t('additionalInfoIcons.description')}
+            exampleFirst
+          >
+            <ComponentExample style={{ justifyContent: 'flex-start' }}>
+              <Icon
+                icon="errorFilled"
+                color={suomifiDesignTokens.colors.highlightBase}
+                style={{ margin: `0 ${suomifiDesignTokens.spacing.s}` }}
+              />
+              <Icon
+                icon="info"
+                color={suomifiDesignTokens.colors.highlightBase}
+                style={{ margin: `0 ${suomifiDesignTokens.spacing.s}` }}
+              />
+            </ComponentExample>
+          </ComponentDescription>
+
+          <ComponentDescription
+            mainTitle={t('illustrationIcons.title')}
+            description={t('illustrationIcons.description')}
+            exampleFirst
+          >
+            <ComponentExample style={{ justifyContent: 'flex-start' }}>
+              <Icon icon="staticIllustrationHands" />
+            </ComponentExample>
+          </ComponentDescription>
+
+          <ComponentDescription
+            mainTitle={t('attachmentIcons.title')}
+            description={t('attachmentIcons.description')}
+            exampleFirst
+          >
+            <ComponentExample style={{ justifyContent: 'flex-start' }}>
+              <Icon icon="doc" />
+            </ComponentExample>
+          </ComponentDescription>
+
+          <ComponentDescription
+            mainTitle={t('signLanguageIcons.title')}
+            description={t('signLanguageIcons.description')}
+            exampleFirst
+          >
+            <ComponentExample style={{ justifyContent: 'flex-start' }}>
+              <Icon icon="signLanguageContent" color="#00B6EC" />
+            </ComponentExample>
+          </ComponentDescription>
```

Comment:
I think elsewhere the icons are presented without the borders in the examples. Could it work here as well?

Reply:
Yeah, we can get rid of borders here.
Design in Abstract was having it so it was left as default.